var/spack/repos/builtin/packages/graphviz/package.py

```diff
@@ -23,16 +23,16 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
+import os
 import sys
-import shutil
 
 
 class Graphviz(AutotoolsPackage):
     """Graph Visualization Software"""
     homepage = 'http://www.graphviz.org'
-    url      = 'http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-2.38.0.tar.gz'
 
-    version('2.38.0', '5b6a829b2ac94efcd5fa3c223ed6d3ae')
+    version('2.40.1', git='https://gitlab.com/graphviz/graphviz.git',
+            tag='stable_release_2.40.1')
```

Comment:
Usually we try to have release versions that are tied to an hash, for security reasons. Does it make sense in this case to use the commit hash instead?

Reply:
Yup!